src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -103,15 +103,15 @@ public function process(ContainerBuilder $container)
             $container->getDefinition('twig.extension.yaml')->addTag('twig.extension');
         }
 
-        if (class_exists(\Symfony\Component\Stopwatch\Stopwatch::class)) {
+        if ($container->isAvailable('symfony/stopwatch', \Symfony\Component\Stopwatch\Stopwatch::class)) {
```

Comment:
this one looks wrong to me. stopwatch is regularly enabled only in dev setups

Reply:
all fixed, thanks